src/lib/tabs/tab-group.spec.ts

```diff
@@ -238,7 +238,7 @@ describe('MatTabGroup', () => {
     });
 
     it('should emit focusChange event on click', () => {
-      spyOn(fixture.componentInstance, 'handleFocus');
+      spyOn(fixture.componentInstance, 'handleFocus')
```

Comment:
```suggestion
      spyOn(fixture.componentInstance, 'handleFocus');
```

Is there any reason why you omitted the semicolon here?

Reply:
Thanks @Chan4077  - no reason, just a missed type.